Source/FullScreenPokemon.ts

```diff
@@ -3832,23 +3832,43 @@ module FullScreenPokemon {
 
         /**
          * Opens the context menu within the Pokedex menu for the selected Pokemon.
+         *
+         * @param settings   The selected Pokemon.
```

Comment:
It's good you're adding the docs here (I must have missed it before).

But, settings itself isn't the selected Pokemon; it _contains_ the Pokemon.